hedyorg/hedy
# [FEATURE] Enables users to view highscores with all class members

**Description**
In this PR we change the design of the highscores slightly. Currently users are only able to view the highscores of users with a public profile. However, in the case of a class some users might not have a public profile while it is still fun to compare the amount of achievements.  We solve this by retrieving achievements of all users when a user visits the class highscores.

**Fixes**
This PR fixes #3638

**How to test**
Make sure you are in a class with users that all have achievements but some might not have a public profile. Verify that these users are still visible when filtering the highscores on class.
